.babelrc

```diff
@@ -1,14 +0,0 @@
-{
```

Comment:
The transform required for the transmute libraries ignores this file... it has been replaced with a `babel.config.js` equivalent instead